vnext/ReactUWP/Base/CoreNativeModules.cpp

```diff
@@ -116,7 +115,7 @@ std::vector<facebook::react::NativeModuleDescription> GetCoreModules(
         if (HasPackageIdentity()) {
```

Comment:
is this right? do we need to be packaged for the storage APIs to work? this seems in opposition to not needing the win32 module. I'm hoping we can always use the WinRT storage APIs whether we're packaged or not

Reply:
I’d have to check with the author of this method.

Can we leave this investigation for a related subsequent PR?

Reply:
I suspect you need to be packaged to get an identity for the storage APIs to know where to store the info.

I'm not too concerned with the functionality here, as AsyncStorage has already been deprecated, and will probably be removed in a release or two.

Reply:
I have worked around this. Won't remove that Win32 module, nor move it into the main project.